kalite/distributed/static/js/distributed/user/views.js

```diff
@@ -362,7 +362,7 @@ window.ToggleNavbarView = BaseView.extend ({
     initialize: function() {
 
         _.bindAll(this);
-        this.listenTo(this.model, "change", this.render);
+        this.listenTo(this.model, "change:client_server_time_diff", this.render);
```

Comment:
How did you choose this event to listen to? I can't figure out when it's triggered.